src/Softeq.NetKit.Chat.Domain/DomainModels/Member.cs

```diff
@@ -19,6 +19,7 @@ public class Member : IBaseEntity<Guid>
         public bool IsAfk { get; set; }
 
         public bool IsActive { get; set; }
+        public bool IsDeleted { get; set; }
```

Comment:
Not sure if it's more proper way for doing this, but let's consider using nullable "DeletedAt" instead. This way we will be able to know if the member is deleted + deletion datetime.